docs/src/examples/components/Popup/Variations/index.tsx

```diff
@@ -9,6 +9,7 @@ const Variations = () => (
       title="Alignment and Position"
       description="A popup can be positioned around its trigger and aligned relative to the trigger's margins. Click on a button to open a popup on a specific position and alignment."
       examplePath="components/Popup/Variations/PopupExamplePosition"
+      previewHtml={false}
```

Comment:
We should also add this on the Tooltip examples as well... A bit hard to keep track of where this should be added. Can't we catch this error and do something afterwards?

Reply:
React doesn't export error codes in errors 👶 

I will try to rethink it 🤔

Reply:
+1 for omitting `previewHtml` and catching error if possible